src/Orchard.Cms.Web/Modules/Orchard.OpenId/Services/OpenIdService.cs

```diff
@@ -15,31 +15,34 @@ namespace Orchard.OpenId.Services
     public class OpenIdService : IOpenIdService
     {
         private readonly ISiteService _siteService;
-        private readonly IMemoryCache _memoryCache;
         private readonly IStringLocalizer<OpenIdService> T;
-        public OpenIdService(ISiteService siteService, IMemoryCache memoryCache, IStringLocalizer<OpenIdService> stringLocalizer)
+
+        public OpenIdService(ISiteService siteService, IStringLocalizer<OpenIdService> stringLocalizer)
         {
             _siteService = siteService;
-            _memoryCache = memoryCache;
             T = stringLocalizer;
         }
+
         public async Task<OpenIdSettings> GetOpenIdSettingsAsync()
         {
-            var siteSettings = await _siteService.GetSiteSettingsAsync();
-            var result = siteSettings.As<OpenIdSettings>();
+            var settings = await _siteService.GetSiteSettingsAsync();
 
+            var result = settings.As<OpenIdSettings>();
             if (result == null)
             {
                 result = new OpenIdSettings();
             }
+
             return result;
         }
+
         public async Task UpdateOpenIdSettingsAsync(OpenIdSettings openIdSettings)
         {
-            var siteSettings = await _siteService.GetSiteSettingsAsync();
-            siteSettings.Properties[nameof(OpenIdSettings)] = JObject.FromObject(openIdSettings);
-            await _siteService.UpdateSiteSettingsAsync(siteSettings);
+            var settings = await _siteService.GetSiteSettingsAsync();
```

Comment:
I see you have made more name changing like this one in vars and in a class name like JwtBearerConfigureOptions.cs per OpenIdConfiguration.cs In general I like to use a variable name that express what it contains. I observe do you prefer more compact names with less meaning. Is it because a rule of orchard development guidelines? Does it respond to a well know good practice? Or is it only a matter of personal taste?